Makefile

```diff
@@ -16,11 +16,15 @@ test-functional:
 test-schemas:
 	pipenv run ./scripts/test_schemas.sh
 
+translate:
```

Comment:
I think we need better namings for these. 
Maybe `make translations` etc

Reply:
I thought about this, but then the translations are really the .po files...